samples/PythonIngestionSample.py

```diff
@@ -0,0 +1,149 @@
+import os
+import json
+import microsoft.bing.commerce.ingestion
+from microsoft.bing.commerce.ingestion import BingCommerceIngestion
+from microsoft.bing.commerce.ingestion.models import IndexField, Index,Region
+from msrest.authentication import BasicTokenAuthentication
+from msrest.exceptions import HttpOperationError
+
+#Creates a definition of the tenant's index. 
+def CreateIndex(self):
+    creds = BasicTokenAuthentication({ 'access_token': os.environ['INGEST_TOKEN'] })
+    client = BingCommerceIngestion(creds)
+    
+    # Create Index (First, make sure we don't have an index with the same name.)
+    field = IndexField(name = 'productId',
+                       type = 'ProductId',
+                       facetable=False,
+                       filterable=False,
+                       retrievable=True,
+                       searchable=True,
+                       sortable=False)
+    
+    new_index = Index(name = 'Contoso',
+                      description = 'Index definition for Contoso.com',
+                      search_scenario='Retail',
+                      regions=[Region.west_us,Region.east_us,Region.north_central_us],
+                      fields = [field])
+
+    create_response = client.create_index(self.TENANT_ID, body=new_index)
```

Comment:
there is no self, this is not in a class -- same for all. Maybe take the tenant id as a param ?